### PR TITLE
feat(bigquery/storage): remove alpha client

### PR DESCRIPTION
**Pull request by codyoss**
This api version should no longer be used.

Updates: #4098

### CODE REVIEW

**Comment by codyoss** [comment]
Created this as a feat so it shows up in release notes

File: internal/gapicgen/generator/config.go [hunk 1]
```diff
@@ -194,15 +194,15 @@ var microgenGapicConfigs = []*microgenConfig{
...
-	{
-		inputDirectoryPath:    "google/cloud/bigquery/storage/v1alpha2",
-		pkg:                   "storage",
-		importPath:            "cloud.google.com/go/bigquery/storage/apiv1alpha2",
-		gRPCServiceConfigPath: "google/cloud/bigquery/storage/v1alpha2/bigquerystorage_grpc_service_config.json",
-		apiServiceConfigPath:  "google/cloud/bigquery/storage/v1alpha2/bigquerystorage_v1alpha2.yaml",
-		releaseLevel:          "alpha",
-		stopGeneration:        true,
-	},
+	// {
```
Comment (noahdietz) [inline internal/gapicgen/generator/config.go:197]: Should this just be removed?

**Review by noahdietz — APPROVED** [review]

File: internal/gapicgen/generator/config.go (hunk 1 above)
Comment (codyoss) [reply internal/gapicgen/generator/config.go]: whoops, yes thanks!

**Review by shollyman — APPROVED** [review]

**Comment by shollyman** [comment]
Seems there's still something related to internal snippets generation?

```
go: finding module for package cloud.google.com/go/bigquery/storage/apiv1alpha2
cloud.google.com/go/internal/generated/bigquery/storage/apiv1alpha2/BigQueryWriteClient/AppendRows imports
	cloud.google.com/go/bigquery/storage/apiv1alpha2: module cloud.google.com/go/bigquery@latest found (v1.18.0, replaced by ../../../bigquery), but does not contain package cloud.google.com/go/bigquery/storage/apiv1alpha2
```

**Pull request merged**